content/topics/web-frontend/react/redux-architecture/_index.md

```diff
@@ -1,5 +1,18 @@
 ---
 _db_id: 515
 content_type: topic
+prerequisites:
+  hard:
+  - topics/redux-intro/
+  soft: []
+ready: true
 title: React + Redux architectural guidelines
----
\ No newline at end of file
+---
+
+Redux is a very powerful state management library used in JavaScript apps to manage the data of your application, and was originally designed for React. Redux helps you write applications that behave consistently by facilitating and sharing data across the components of an application.
+
+### Fundamental principles
```

Comment:
This should be a level 2 heading.

Reply:
And I dont think you are actually covering any architectural guidelines here...

If you have content that should be added to topics/redux-intro/ then it should go there, not here. 

Eg getting started with redux is definately introductiory material. It has little to do with scalable architectures. Same can be said for "thinking in redux"